### PR TITLE
Wildcards(BreakoutProcess.md)

### PR DESCRIPTION
**Pull request by Luro223**
Added several info describing wildcards * and ?

### CODE REVIEW

**Review by Luro223 — COMMENTED** [review]
Additional changes

File: Content/BreakoutProcess.md [hunk 1]
```diff
@@ -10,8 +10,13 @@ Usage:
    .
    [DefaultBox]
    BreakoutProcess=ProgramName.exe
+   BreakoutProcess=Program*.exe
+   BreakoutProcess=Program?.exe
 ```
 
+ `*` defines any name after Program(Program0Test1.exe, Program5Test92G.exe and etc.).
```
Comment (isaak654) [inline Content/BreakoutProcess.md]:
```suggestion
 `*` defines any name after Program (Program0Test1.exe, Program5Test92G.exe and etc.).
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Review by isaak654 — APPROVED** [review]

File: Content/BreakoutProcess.md [hunk 2]
```diff
@@ -10,8 +10,13 @@ Usage:
    .
    [DefaultBox]
    BreakoutProcess=ProgramName.exe
+   BreakoutProcess=Program*.exe
+   BreakoutProcess=Program?.exe
 ```
 
+ `*` defines any name after Program (Program0Test1.exe, Program5Test92G.exe and etc.).
+ `?` defines 1 character from name (Program1.exe, Program2.exe .. Program9.exe).
```
Comment (isaak654) [inline Content/BreakoutProcess.md]:
```suggestion
 `?` defines one character from name (Program1.exe, Program2.exe .. Program9.exe).
```

File: Content/BreakoutProcess.md [hunk 3]
```diff
@@ -10,8 +10,16 @@ Usage:
...
    BreakoutProcess=ProgramName.exe
+   BreakoutProcess=Program*.exe
+   BreakoutProcess=Program?.exe
+   BreakoutProcess=Pro?ram*.exe
 ```
 
+ `*` defines any name after Program (Program0Test1.exe, Program5Test92G.exe and etc.).
+ `?` defines one character from name (Program1.exe, Program2.exe .. Program9.exe).
+ 
+Also you can combine several wildcards to match the specified name
```
Comment (isaak654) [inline Content/BreakoutProcess.md]:
```suggestion
Also, you can combine several wildcards to match the specified name.
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Review by isaak654 — APPROVED** [review]

**Review by isaak654 — APPROVED** [review]

**Pull request merged**